editor/src/components/canvas/canvas-strategies/strategies/set-padding-strategy.tsx

```diff
@@ -246,11 +245,70 @@ export const setPaddingStrategy: CanvasStrategyFactory = (canvasState, interacti
 
       // "tearing off" padding
       if (newPaddingEdge.renderedValuePx < PaddingTearThreshold) {
-        return strategyApplicationResult([
+        return strategyApplicationResult(
+          [
+            ...basicCommands,
+            deleteProperties('always', selectedElement, [
+              StylePaddingProp,
+              stylePropPathMappingFn(paddingPropInteractedWith, styleStringInArray),
+            ]),
+            ...nonZeroPropsToAdd.map(([p, value]) =>
+              setProperty(
+                'always',
+                selectedElement,
+                stylePropPathMappingFn(p, styleStringInArray),
+                value,
+              ),
+            ),
+            setActiveFrames(
+              selectedElements.map((path) => ({
+                action: 'set-padding',
+                target: activeFrameTargetPath(path),
+                source: zeroRectIfNullOrInfinity(
+                  MetadataUtils.getFrameInCanvasCoords(path, canvasState.startingMetadata),
+                ),
+              })),
+            ),
+          ],
+          'rerender-all-elements',
+        )
+      }
+
+      const allPaddingPropsDefined = maybeFullPadding(newPaddingMaxed)
+
+      // all 4 sides present - can be represented via the padding shorthand property
+      if (allPaddingPropsDefined != null) {
+        const paddingString = paddingToPaddingString(allPaddingPropsDefined)
+        return strategyApplicationResult(
+          [
+            ...basicCommands,
+            ...IndividualPaddingProps.map((p) =>
+              deleteProperties('always', selectedElement, [
+                stylePropPathMappingFn(p, styleStringInArray),
+              ]),
+            ),
+            setProperty('always', selectedElement, StylePaddingProp, paddingString),
+            setActiveFrames(
+              selectedElements.map((path) => ({
+                action: 'set-padding',
+                target: activeFrameTargetPath(path),
+                source: zeroRectIfNullOrInfinity(
+                  MetadataUtils.getFrameInCanvasCoords(path, canvasState.startingMetadata),
+                ),
+              })),
+            ),
+          ],
+          'rerender-all-elements',
```

Comment:
possibly one to fix here too

Reply:
Added comment about follow up in 38ec355d5.